### ISSUE TITLE
Native nesting

### ISSUE THREAD

**Issue by jxpx777** [amadeus/vim-css#5]
I'm working on a personal project in order to learn more current CSS and discovered that nesting is now [supported](https://caniuse.com/css-nesting) in very recent releases of major browsers but I noticed syntax errors in my CSS file when using nesting. From the syntax file, it found that I can set `b:embedded_rules=1` and get the errors to disappear, but I'm wondering if this can simply be removed now that nesting is a native feature of CSS.

**Comment by amadeus** [comment]
Yeah this something probably worth revisiting, but given there's a work around probably won't look into it anytime soon
